src/braket/experimental/autoqasm/instructions/measurements.py

```diff
@@ -42,7 +46,7 @@ def measure(qubits: Union[QubitIdentifierType, Iterable[QubitIdentifierType]]) -
     Returns:
         BitVar: Bit variable the measurement results are assigned to.
     """
-    if not isinstance(qubits, Iterable):
+    if is_qubit_identifier_type(qubits):
```

Comment:
Nice solution!

Reply:
I always seem to forget that `str` is an `Iterable`